vignettes/hubverse_ms_vignette.Rmd

```diff
@@ -0,0 +1,220 @@
+---
+title: "Supplemental Code Demo"
+subtitle: "Accessing Data from the US Flusight Hubverse Project"
+author: 
+- Consortium of Infectious Disease Modeling Hubs
+- Melissa Kerr
+- Rebecca Borchering
+- Alvaro Castro Rivadeneira
+- Lucie Contamin
+- Sebastian Funk
+- Harry Hochheiser
+- Emily Howerton
+- Anna Krystalli
+- Li Shandross
+- Nicholas G Reich
+date: "`r Sys.Date()`"
+output: 
+  pdf_document:
+    citation_package: natbib
+bibliography: references.bib
+---
+
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  warning = FALSE
+)
+```
+
+# Software configuration
+The following demonstration shows how to access and analyze data from a real hub using `r R.version$string` and several of the hubverse R packages.[@the_consortium_of_infectious_disease_modeling_hubs_hubverse_2025] The following packages are on CRAN and may be installed using the standard `install.packages()` function: `hubEnsembles`, `aws.s3`, `arrow`, `ggplot2` and `dplyr`.
+
+The hubverse packages `hubData`, `hubVis`, and `hubEvals` must be installed through the R-universe package repository using the commands below.
+```{r install, eval=FALSE}
+install.packages(pkgs = c("hubData", "hubVis", "hubEvals"),
+                 repos = c("https://hubverse-org.r-universe.dev",
+                           "https://cloud.r-project.org"))
+```
+
+Once the required packages are installed, load them into your R session. (Session information including package versions are included at the end of this vignette.)
+
+```{r libraries, message=FALSE}
+library(hubData)
+library(hubEnsembles)
+library(hubVis)
+library(hubEvals)
+```
+
+```{r, echo=FALSE}
+ggplot2::theme_set(ggplot2::theme_bw())
+```
+
+
+# Load forecast data
+CDC's FluSight Forecast Hub exists as a [GitHub repository](https://github.com/cdcepi/FluSight-forecast-hub).[@noauthor_cdcepiflusight-forecast-hub_2025] Using hubverse tools, it also has been mirrored to the cloud, so a copy of the data lives at a publicly-accessible AWS S3 bucket. This means that instead of needing to download a local copy of the repository, the data for this project can be directly accessed from the cloud using `hubData` tools.
+
```

Comment:
Should we add a little more context to the hub, saying what the goal is quickly and that the data mirrored on the bucket have all passed validation (using hubValidations)